code/modules/research/techweb/all_nodes.dm

```diff
@@ -738,6 +738,7 @@
 		"dispenser_shell",
 		"door_shell",
 		"gun_shell",
+		"keyboard_shell",
```

Comment:
needs to be placed in alphabetical order in line with everything else

Reply:
I think it's already in alphabetical order. Am I missing something ?

Reply:
me dumb